.gitignore

```diff
@@ -6,3 +6,4 @@
 # Django
 *.pyc
 *.sqlite3
+media
```

Comment:
What's a media file?

Reply:
It seems to just hold locally stored data such as images. When I was testing email confirmation along with using a user's preferred names in the email, I needed to create dummy data and doing so required me to give the data an image, which then created a media folder, with an image folder inside it, holding the image folder that I placed.

Basically this is just to ignore those files.